tmpl/blog/2016-spring-hackathon.md

```diff
@@ -0,0 +1,231 @@
+# Hackathon Trip Reports
+
+We're looking forward to the next MirageOS hackathon already!  We've collected some reports from those who were present.  Thanks to the folks who put in the time and effort to organize the event and our wonderful hosts, and a huge thanks to everyone who documented their hackathon experience!
```

Comment:
hmm, ok... I had the hope I could silently destroy the marrakech2016.mirage.io unikernel now that the hackathon is over... ;)